src/main/java/io/galeb/manager/entity/BalancePolicy.java

```diff
@@ -79,4 +79,9 @@ public BalancePolicy setPools(Set<Pool> pools) {
         }
         return this;
     }
+
+    @Override
+    public String getEnvName() {
+        return null;
```

Comment:
There's a NullPointerException risk. Using generic constant may be a best approach, not?